subsys/bluetooth/host/Kconfig

```diff
@@ -142,9 +142,6 @@ config BT_RX_STACK_SIZE
 	default 2048 if BT_MESH
 	default 2200 if BT_SETTINGS
 	default 1024
-	range 512 65536 if BT_HCI_RAW
-	range 1100 65536 if BT_MESH
-	range 1024 65536
```

Comment:
Shouldn't we still have a minimal for BT_MESH and BT_HCI_RAW, if we just depend on the default the application may break the stack if it sets the value too low or is there something preventing that to happen? Maybe we need to check what is the actul minimal we need for the RX thread to work properly.

Reply:
That is what I want to avoid. The minimum depends on the use-case since the application has callbacks given from the RX thread.

This kconfig is used in two different places as mentioned here: https://github.com/zephyrproject-rtos/zephyr/pull/29279#issuecomment-710101800

I am fine with letting the default stack sizes be inefficient. But I need to have tight minimal range in order to optimize the memory usage as much as possible.
The application has the responsibility to tune it in any case due to the callbacks, so removing the range saves me the trouble of finding a lot of minimal values that wouldn't be very trustworthy in any case.